std/experimental/logger/core.d

```diff
@@ -1,17 +1,9 @@
 ///
 module std.experimental.logger.core;
 
-import std.array;
-import std.stdio;
-import std.conv;
 import std.datetime;
-import std.string;
-import std.range;
+import std.range.primitives;
 import std.traits;
```

Comment:
There's really no benefit to top-level selective imports at the moment. DMD still scans the entire imported file; all selective imports do is change visibility rules. This is something that can and should change (SDC does the right thing), and when it does, I will gladly fix this.

Plus, selective top level imports aren't the current style in the rest of Phobos.